mlflow/server/handlers.py

```diff
@@ -1230,6 +1230,70 @@ def assert_arg_exists(arg_name, arg):
     return response
 
 
+@catch_mlflow_exception
+def upload_artifact_handler():
+    _logger.info("upload_artifact_handler")
+    args = request.args
+    run_uuid = args.get("run_uuid")
+    if not run_uuid:
+        raise MlflowException(
+            message="Request must specify run_uuid.",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
+    path = args.get("path")
+    if not path:
```

Comment:
Is an empty string valid for `path`?

Reply:
No. Empty string for path should throw. I also call `validate_path_is_safe` to be safe.